packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -414,6 +414,10 @@ access_token:
     expires: never
     data_sensitivity:
       - interaction
+    extra_keys:
+      scopes:
+        description: The scopes of the access token
+        type: string
```

Comment:
Does Glean support Array types? Would that allow for easier sorting and graphs?

Reply:
Unfortunately, the String List type is not support for the  `extra keys`.  😞